Upgrading from <8.5.0 to 8.13.0 fails on MacOS

Upgrading from 7.17.x to 8.13.0 reports a syntax error in the elastic agent script, eventually resulting in a rollback to the previous version. (Upgrading the same configuration to 8.12.2 is successful, as is upgrading 8.11.1 to 8.13.0):

- Set the fleet artifacts path to the 8.13 build candidate URL
- Create an Agent policy with the System and Defend integrations
- Install and enroll Agent 7.17.x with the policy [reproduced with both 7.17.16 and 7.17.18, not sure the full range]
- Upgrade the agent, manually entering version `8.13.0`

The Agent will go from "upgrading" to "healthy" while remaining at the old version.

During the upgrade, a shell error is reported while checking the elastic-agent version:

```sh
> sudo elastic-agent version
Binary: 7.17.18 (build: aba352d2bdba2e2ab7da09beb505e772ed540f88 at 2024-02-02 14:28:02 +0000 UTC)
Daemon: 7.17.18 (build: aba352d2bdba2e2ab7da09beb505e772ed540f88 at 2024-02-02 14:28:02 +0000 UTC)

...

> sudo elastic-agent version
/Library/Elastic/Agent/elastic-agent: line 15: syntax error: unexpected end of file

...

> sudo elastic-agent version
Binary: 7.17.18 (build: aba352d2bdba2e2ab7da09beb505e772ed540f88 at 2024-02-02 14:28:02 +0000 UTC)
Daemon: 7.17.18 (build: aba352d2bdba2e2ab7da09beb505e772ed540f88 at 2024-02-02 14:28:02 +0000 UTC)
```

The contents of `/Library/Elastic/Agent/elastic-agent` while it is reporting a syntax error are:

```sh
#!/bin/sh
# Fix up the symlink and exit

set -e

symlink="/Library/Elastic/Agent/elastic-agent"

if test -L "$symlink"; then
    symlinkTarget="data/elastic-agent-533443/elastic-agent.app/Contents/MacOS/elastic-agent"
    if test -f "data/elastic-agent-8.13.0-533443/elastic-agent.app/Contents/MacOS/elastic-agent"; then
        symlinkTarget="data/elastic-agent-8.13.0-533443/elastic-agent.app/Contents/MacOS/elastic-agent"
    ln -sfn "$symlinkTarget" "$symlink"
fi
```

The logs reported in Fleet during the failed upgrade (prerelease artifacts URL redacted) are:


<details>

<summary>upgrade logs</summary>

```
Showing entries from Mar 21, 12:11:46
12:11:46.734
elastic_agent
[elastic_agent][info] Detecting execution mode
12:11:46.734
elastic_agent
[elastic_agent][info] Detecting execution mode
12:11:46.734
elastic_agent
[elastic_agent][info] Detecting execution mode
12:11:46.736
elastic_agent
[elastic_agent][info] Agent is managed locally
12:11:46.736
elastic_agent
[elastic_agent][info] Agent is managed locally
12:11:46.736
elastic_agent
[elastic_agent][info] Agent is managed locally
12:11:46.737
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:11:46.737
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:11:46.737
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:11:46.993
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:11:46.993
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:11:46.993
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:11:46.995
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:11:46.995
elastic_agent
[elastic_agent][info] Agent is starting
12:11:46.995
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:11:46.995
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:11:46.995
elastic_agent
[elastic_agent][info] Agent is starting
12:11:46.995
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:11:46.995
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:11:46.995
elastic_agent
[elastic_agent][info] Agent is starting
12:11:46.995
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:11:47.000
elastic_agent
[elastic_agent][info] Agent is stopped
12:11:47.000
elastic_agent
[elastic_agent][info] Agent is stopped
12:11:47.000
elastic_agent
[elastic_agent][info] Agent is stopped
12:11:47.001
elastic_agent
[elastic_agent][info] Configuration changes detected
12:11:47.001
elastic_agent
[elastic_agent][info] Configuration changes detected
12:11:47.001
elastic_agent
[elastic_agent][info] Configuration changes detected
12:11:47.013
elastic_agent
[elastic_agent][info] Source URI reset from "https://artifacts.elastic.co/downloads/" to "https://artifacts.elastic.co/downloads/"
12:11:47.013
elastic_agent
[elastic_agent][info] Source URI reset from "https://artifacts.elastic.co/downloads/" to "https://artifacts.elastic.co/downloads/"
12:11:47.013
elastic_agent
[elastic_agent][info] Source URI reset from "https://artifacts.elastic.co/downloads/" to "https://artifacts.elastic.co/downloads/"
12:11:47.014
elastic_agent
[elastic_agent][info] New State ID is TyehGjzZ
12:11:47.014
elastic_agent
[elastic_agent][info] New State ID is TyehGjzZ
12:11:47.014
elastic_agent
[elastic_agent][info] New State ID is TyehGjzZ
12:11:47.015
elastic_agent
[elastic_agent][info] Converging state requires execution of 2 step(s)
12:11:47.015
elastic_agent
[elastic_agent][info] Converging state requires execution of 2 step(s)
12:11:47.015
elastic_agent
[elastic_agent][info] Converging state requires execution of 2 step(s)
12:11:48.799
elastic_agent
[elastic_agent][info] waiting for installer of pipeline 'default' to finish
12:11:48.799
elastic_agent
[elastic_agent][info] waiting for installer of pipeline 'default' to finish
12:11:48.799
elastic_agent
[elastic_agent][info] waiting for installer of pipeline 'default' to finish
12:11:48.868
elastic_agent
[elastic_agent][info] No configuration change
12:11:48.868
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18
12:11:48.868
elastic_agent
[elastic_agent][info] No configuration change
12:11:48.868
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18
12:11:48.868
elastic_agent
[elastic_agent][info] No configuration change
12:11:48.868
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18
12:11:48.871
elastic_agent
[elastic_agent][info] Source URI reset from "https://artifacts.elastic.co/downloads/" to "https://artifacts.elastic.co/downloads/"
12:11:48.871
elastic_agent
[elastic_agent][info] Source URI reset from "https://artifacts.elastic.co/downloads/" to "https://artifacts.elastic.co/downloads/"
12:11:48.871
elastic_agent
[elastic_agent][info] Source URI reset from "https://artifacts.elastic.co/downloads/" to "https://artifacts.elastic.co/downloads/"
12:11:50.513
elastic_agent
[elastic_agent][info] Detecting execution mode
12:11:50.513
elastic_agent
[elastic_agent][info] Detecting execution mode
12:11:50.513
elastic_agent
[elastic_agent][info] Detecting execution mode
12:11:50.516
elastic_agent
[elastic_agent][info] Agent is managed by Fleet
12:11:50.516
elastic_agent
[elastic_agent][info] Agent is managed by Fleet
12:11:50.516
elastic_agent
[elastic_agent][info] Agent is managed by Fleet
12:11:50.517
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:11:50.517
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:11:50.517
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:11:50.912
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:11:50.912
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:11:50.912
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:11:50.914
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:11:50.914
elastic_agent
[elastic_agent][info] Agent is starting
12:11:50.914
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:11:50.914
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:11:50.914
elastic_agent
[elastic_agent][info] Agent is starting
12:11:50.914
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:11:50.914
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:11:50.914
elastic_agent
[elastic_agent][info] Agent is starting
12:11:50.914
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:11:52.902
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:11:52.902
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:11:52.902
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:11:52.904
elastic_agent
[elastic_agent][info] New State ID is xZ_PQneS
12:11:52.904
elastic_agent
[elastic_agent][info] Converging state requires execution of 4 step(s)
12:11:52.904
elastic_agent
[elastic_agent][info] New State ID is xZ_PQneS
12:11:52.904
elastic_agent
[elastic_agent][info] Converging state requires execution of 4 step(s)
12:11:52.904
elastic_agent
[elastic_agent][info] New State ID is xZ_PQneS
12:11:52.904
elastic_agent
[elastic_agent][info] Converging state requires execution of 4 step(s)
12:11:55.944
elastic_agent
[elastic_agent][info] 2024-03-21T12:11:55-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:11:55.944
elastic_agent
[elastic_agent][info] 2024-03-21T12:11:55-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:11:55.944
elastic_agent
[elastic_agent][info] 2024-03-21T12:11:55-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:29.204
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:29-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:29.204
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:29-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:29.204
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:29-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:36.499
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:36-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to CONFIG: Updating configuration - type: 'STATE' - sub_type: 'CONFIG'
12:12:36.499
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:36-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to CONFIG: Updating configuration - type: 'STATE' - sub_type: 'CONFIG'
12:12:36.499
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:36-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to CONFIG: Updating configuration - type: 'STATE' - sub_type: 'CONFIG'
12:12:40.495
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:40-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Protecting with policy {ed800426-bd73-4386-bf4f-6fc7aecddb32} - type: 'STATE' - sub_type: 'RUNNING'
12:12:40.495
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:40-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Protecting with policy {ed800426-bd73-4386-bf4f-6fc7aecddb32} - type: 'STATE' - sub_type: 'RUNNING'
12:12:40.495
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:40-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Protecting with policy {ed800426-bd73-4386-bf4f-6fc7aecddb32} - type: 'STATE' - sub_type: 'RUNNING'
12:12:53.328
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:53-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:53.328
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:53-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:53.328
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:53-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:53.565
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:12:53.565
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:12:53.565
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:12:53.663
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:53-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:53.663
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:53-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:53.663
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:53-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:53.829
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:12:53.829
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:12:53.829
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:12:54.032
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:54-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:54.032
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:54-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:54.032
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:54-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:12:54.037
elastic_agent
[elastic_agent][info] Updating internal state
12:12:54.037
elastic_agent
[elastic_agent][info] Updating internal state
12:12:54.037
elastic_agent
[elastic_agent][info] Updating internal state
12:12:54.841
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:54-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:54.841
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:54-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:54.841
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:54-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:57.691
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:57-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:57.691
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:57-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:57.691
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:57-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:57.691
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:57-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:57.691
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:57-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:12:57.691
elastic_agent
[elastic_agent][info] 2024-03-21T12:12:57-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:14:48.980
elastic_agent
[elastic_agent][info] 2024-03-21T12:14:48-04:00 - message: Application: [a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to UPDATING: Update to version '8.13.0' started - type: 'STATE' - sub_type: 'UPDATING'
12:14:48.980
elastic_agent
[elastic_agent][info] 2024-03-21T12:14:48-04:00 - message: Application: [a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to UPDATING: Update to version '8.13.0' started - type: 'STATE' - sub_type: 'UPDATING'
12:14:48.980
elastic_agent
[elastic_agent][info] 2024-03-21T12:14:48-04:00 - message: Application: [a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to UPDATING: Update to version '8.13.0' started - type: 'STATE' - sub_type: 'UPDATING'
12:15:05.636
elastic_agent
[elastic_agent][info] download from [redacted]beats/elastic-agent/elastic-agent-8.13.0-darwin-x86_64.tar.gz completed in 16 seconds @ 22.58MBps
12:15:05.636
elastic_agent
[elastic_agent][info] download from [redacted]beats/elastic-agent/elastic-agent-8.13.0-darwin-x86_64.tar.gz completed in 16 seconds @ 22.58MBps
12:15:05.636
elastic_agent
[elastic_agent][info] download from [redacted]beats/elastic-agent/elastic-agent-8.13.0-darwin-x86_64.tar.gz completed in 16 seconds @ 22.58MBps
12:15:05.784
elastic_agent
[elastic_agent][info] download from [redacted]beats/elastic-agent/elastic-agent-8.13.0-darwin-x86_64.tar.gz.sha512 completed in Less than a second @ +InfYBps
12:15:05.784
elastic_agent
[elastic_agent][info] download from [redacted]beats/elastic-agent/elastic-agent-8.13.0-darwin-x86_64.tar.gz.sha512 completed in Less than a second @ +InfYBps
12:15:05.784
elastic_agent
[elastic_agent][info] download from [redacted]beats/elastic-agent/elastic-agent-8.13.0-darwin-x86_64.tar.gz.sha512 completed in Less than a second @ +InfYBps
12:15:16.067
elastic_agent
[elastic_agent][info] waiting for installer of pipeline 'default' to finish
12:15:16.067
elastic_agent
[elastic_agent][info] waiting for installer of pipeline 'default' to finish
12:15:16.068
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:16-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:16.068
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: filebeat--7.17.18
12:15:16.068
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:16-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:16.068
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: filebeat--7.17.18
12:15:17.307
elastic_agent
[elastic_agent][error] context canceled
12:15:17.307
elastic_agent
[elastic_agent][error] context canceled
12:15:17.587
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18
12:15:17.587
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:17-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:17.587
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18
12:15:17.587
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:17-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:18.322
elastic_agent
[elastic_agent][error] context canceled
12:15:18.322
elastic_agent
[elastic_agent][error] context canceled
12:15:19.110
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: filebeat--7.17.18--36643631373035623733363936343635
12:15:19.110
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:19-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:19.110
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: filebeat--7.17.18--36643631373035623733363936343635
12:15:19.110
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:19-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:19.330
elastic_agent
[elastic_agent][info] Fleet gateway is stopping
12:15:19.330
elastic_agent
[elastic_agent][info] Fleet gateway is stopping
12:15:20.623
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18--36643631373035623733363936343635
12:15:20.623
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:20-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:20.623
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18--36643631373035623733363936343635
12:15:20.623
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:20-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:22.135
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:22-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:22.135
elastic_agent
[elastic_agent][info] Agent is stopped
12:15:22.135
elastic_agent
[elastic_agent][info] 2024-03-21T12:15:22-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:15:22.135
elastic_agent
[elastic_agent][info] Agent is stopped
12:17:53.244
elastic_agent
[elastic_agent][info] Detecting execution mode
12:17:53.244
elastic_agent
[elastic_agent][info] Detecting execution mode
12:17:53.246
elastic_agent
[elastic_agent][info] Agent is managed by Fleet
12:17:53.246
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:17:53.246
elastic_agent
[elastic_agent][info] Agent is managed by Fleet
12:17:53.246
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:17:53.479
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:17:53.479
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:17:53.481
elastic_agent
[elastic_agent][info] restoring current policy from disk
12:17:53.481
elastic_agent
[elastic_agent][info] restoring current policy from disk
12:17:53.492
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:17:53.492
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:17:53.493
elastic_agent
[elastic_agent][info] New State ID is 4jkC97zO
12:17:53.493
elastic_agent
[elastic_agent][info] Converging state requires execution of 4 step(s)
12:17:53.493
elastic_agent
[elastic_agent][info] New State ID is 4jkC97zO
12:17:53.493
elastic_agent
[elastic_agent][info] Converging state requires execution of 4 step(s)
12:17:53.636
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:53.636
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:53.700
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:53-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:53.700
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:53-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:55.316
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for endpoint-security.7.17.18
12:17:55.316
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for endpoint-security.7.17.18
12:17:55.317
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:55-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:55.317
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:55-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:55.616
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:55.616
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:55.866
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:55-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:55.866
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:55-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:55.994
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:55.994
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:56.085
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:56-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:56.085
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:56-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:56.236
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:56.236
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:56.351
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:56-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:56.351
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:56-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:17:56.355
elastic_agent
[elastic_agent][info] Updating internal state
12:17:56.355
elastic_agent
[elastic_agent][info] Updating internal state
12:17:56.362
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:17:56.362
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:17:56.363
elastic_agent
[elastic_agent][info] Source URI in operator changed to "[redacted]"
12:17:56.363
elastic_agent
[elastic_agent][info] Source URI in operator changed to "[redacted]"
12:17:56.364
elastic_agent
[elastic_agent][info] New State ID is 4jkC97zO
12:17:56.364
elastic_agent
[elastic_agent][info] Converging state requires execution of 3 step(s)
12:17:56.364
elastic_agent
[elastic_agent][info] New State ID is 4jkC97zO
12:17:56.364
elastic_agent
[elastic_agent][info] Converging state requires execution of 3 step(s)
12:17:56.511
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:56.511
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for metricbeat.7.17.18
12:17:56.511
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:56.511
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for metricbeat.7.17.18
12:17:56.632
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:56.632
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for filebeat.7.17.18
12:17:56.632
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:56.632
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for filebeat.7.17.18
12:17:56.751
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:56.751
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for filebeat.7.17.18
12:17:56.751
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:17:56.751
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for filebeat.7.17.18
12:17:56.898
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:56.898
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for metricbeat.7.17.18
12:17:56.898
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:17:56.898
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for metricbeat.7.17.18
12:17:56.900
elastic_agent
[elastic_agent][info] Updating internal state
12:17:56.900
elastic_agent
[elastic_agent][info] Updating internal state
12:17:57.043
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:17:57.043
elastic_agent
[elastic_agent][info] Agent is starting
12:17:57.043
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:17:57.043
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:17:57.043
elastic_agent
[elastic_agent][info] Agent is starting
12:17:57.043
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:17:57.856
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:57-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:17:57.856
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:57-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:17:58.475
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:58-04:00 - message: Application: [a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING:  - type: 'STATE' - sub_type: 'RUNNING'
12:17:58.475
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:58-04:00 - message: Application: [a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING:  - type: 'STATE' - sub_type: 'RUNNING'
12:17:59.966
elastic_agent
[elastic_agent][info] 2024-03-21T12:17:59-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to CONFIG: Updating configuration - type: 'STATE' - sub_type: 'CONFIG'
12:18:00.178
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:00-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:18:00.482
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:00-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:18:01.132
elastic_agent
[elastic_agent][info] waiting for installer of pipeline 'default' to finish
12:18:01.132
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18--36643631373035623733363936343635
12:18:01.132
elastic_agent
[elastic_agent][error] Could not communicate with fleet-server Checking API will retry, error: fail to checkin to fleet-server: Post "https://d67e6331f1cd47d4b9f9a70f02f318c3.fleet.us-west2.gcp.elastic-cloud.com:443/api/fleet/agents/a066e8e7-069d-45f9-9e89-e5d7176b447a/checkin?": context canceled
12:18:01.645
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: metricbeat--7.17.18
12:18:01.645
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:01-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:18:03.155
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: filebeat--7.17.18
12:18:03.155
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:03-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:18:03.155
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:03-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:18:04.667
elastic_agent
[elastic_agent][info] Signaling application to stop because of shutdown: filebeat--7.17.18--36643631373035623733363936343635
12:18:04.667
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:04-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:18:05.180
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:05-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STOPPED: Stopped - type: 'STATE' - sub_type: 'STOPPED'
12:18:05.180
elastic_agent
[elastic_agent][info] Agent is stopped
12:18:05.363
elastic_agent
[elastic_agent][info] Detecting execution mode
12:18:05.365
elastic_agent
[elastic_agent][info] Agent is managed by Fleet
12:18:05.365
elastic_agent
[elastic_agent][info] capabilities file not found in /Library/Elastic/Agent/capabilities.yml
12:18:05.647
elastic_agent
[elastic_agent][info] Docker provider skipped, unable to connect: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
12:18:05.650
elastic_agent
[elastic_agent][info] restoring current policy from disk
12:18:05.662
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:18:05.663
elastic_agent
[elastic_agent][info] New State ID is 4jkC97zO
12:18:05.663
elastic_agent
[elastic_agent][info] Converging state requires execution of 4 step(s)
12:18:05.812
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:18:06.019
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:06-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:18:06.382
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for endpoint-security.7.17.18
12:18:06.383
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:06-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:18:06.722
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:18:06.866
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:06-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:18:07.015
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:18:07.174
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:07-04:00 - message: Application: metricbeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:18:07.290
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:07-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:18:07.443
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:18:07.562
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:07-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to STARTING: Starting - type: 'STATE' - sub_type: 'STARTING'
12:18:07.568
elastic_agent
[elastic_agent][info] Updating internal state
12:18:07.576
elastic_agent
[elastic_agent][info] Source URI changed from "[redacted]" to "[redacted]"
12:18:07.576
elastic_agent
[elastic_agent][info] Source URI in operator changed to "[redacted]"
12:18:07.578
elastic_agent
[elastic_agent][info] New State ID is 4jkC97zO
12:18:07.578
elastic_agent
[elastic_agent][info] Converging state requires execution of 2 step(s)
12:18:07.701
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:18:07.701
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for filebeat.7.17.18
12:18:07.828
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for filebeat.7.17.18
12:18:07.828
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for filebeat.7.17.18
12:18:07.983
elastic_agent
[elastic_agent][info] operation 'operation-install' skipped for metricbeat.7.17.18
12:18:07.983
elastic_agent
[elastic_agent][info] operation 'operation-start' skipped for metricbeat.7.17.18
12:18:07.985
elastic_agent
[elastic_agent][info] Updating internal state
12:18:08.080
elastic_agent
[elastic_agent][info] Starting stats endpoint
12:18:08.080
elastic_agent
[elastic_agent][info] Agent is starting
12:18:08.080
elastic_agent
[elastic_agent][info] Metrics endpoint listening on: /Library/Elastic/Agent/data/tmp/elastic-agent.sock (configured: unix:///Library/Elastic/Agent/data/tmp/elastic-agent.sock)
12:18:16.097
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:16-04:00 - message: Application: filebeat--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to CONFIG: Updating configuration - type: 'STATE' - sub_type: 'CONFIG'
12:18:16.104
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:16-04:00 - message: Application: filebeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:18:16.161
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:16-04:00 - message: Application: metricbeat--7.17.18--36643631373035623733363936343635[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Running - type: 'STATE' - sub_type: 'RUNNING'
12:18:34.221
elastic_agent
[elastic_agent][warn] Elastic Agent status changed to: 'degraded'
12:18:34.221
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:34-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to DEGRADED: Protecting with policy {ed800426-bd73-4386-bf4f-6fc7aecddb32} - type: 'STATE' - sub_type: 'RUNNING'
12:18:35.742
elastic_agent
[elastic_agent][info] Elastic Agent status changed to: 'online'
12:18:35.742
elastic_agent
[elastic_agent][info] 2024-03-21T12:18:35-04:00 - message: Application: endpoint-security--7.17.18[a066e8e7-069d-45f9-9e89-e5d7176b447a]: State changed to RUNNING: Protecting with policy {ed800426-bd73-4386-bf4f-6fc7aecddb32} - type: 'STATE' - sub_type: 'RUNNING'
Showing entries until Mar 21, 12:18:35
```

</details>

```sh
❯ cat ~/Downloads/script.sh
#!/bin/sh
# Fix up the symlink and exit

set -e

symlink="/Library/Elastic/Agent/elastic-agent"

if test -L "$symlink"; then
    symlinkTarget="data/elastic-agent-533443/elastic-agent.app/Contents/MacOS/elastic-agent"
    if test -f "data/elastic-agent-8.13.0-533443/elastic-agent.app/Contents/MacOS/elastic-agent"; then
        symlinkTarget="data/elastic-agent-8.13.0-533443/elastic-agent.app/Contents/MacOS/elastic-agent"
    ln -sfn "$symlinkTarget" "$symlink"
fi%

❯ shellcheck ~/Downloads/script.sh

In /Users/cmackenzie/Downloads/script.sh line 8:
if test -L "$symlink"; then
^-- SC1046 (error): Couldn't find 'fi' for this 'if'.
^-- SC1073 (error): Couldn't parse this if expression. Fix to allow more checks.


In /Users/cmackenzie/Downloads/script.sh line 13:
fi
  ^-- SC1047 (error): Expected 'fi' matching previously mentioned 'if'.
  ^-- SC1072 (error): Expected 'fi'. Fix any mentioned problems and try again.

For more information:
  https://www.shellcheck.net/wiki/SC1046 -- Couldn't find 'fi' for this 'if'.
  https://www.shellcheck.net/wiki/SC1047 -- Expected 'fi' matching previously...
  https://www.shellcheck.net/wiki/SC1072 -- Expected 'fi'. Fix any mentioned ...
```

--- a/dev-tools/packaging/templates/darwin/elastic-agent.tmpl
+++ b/dev-tools/packaging/templates/darwin/elastic-agent.tmpl
@@ -9,6 +9,7 @@ if test -L "$symlink"; then
     symlinkTarget="data/elastic-agent-{{ commit_short }}/elastic-agent.app/Contents/MacOS/elastic-agent"
     if test -f "data/elastic-agent-{{ agent_package_version }}{{ snapshot_suffix }}-{{ commit_short }}/elastic-agent.app/Contents/MacOS/elastic-agent"; then
         symlinkTarget="data/elastic-agent-{{ agent_package_version }}{{ snapshot_suffix }}-{{ commit_short }}/elastic-agent.app/Contents/MacOS/elastic-agent"
+    fi
     ln -sfn "$symlinkTarget" "$symlink"
 fi
 

